binder/views.py

```diff
@@ -1202,6 +1211,12 @@ def _multi_put_save_objects(self, ordered_objects, objects, request):
 				validation_errors.append(e)
 			if oid < 0:
 				new_id_map[(model, oid)] = obj.id
+				for base in getmro(model)[1:]:
+					if not (
+						hasattr(base, 'Meta') and
+						getattr(base.Meta, 'abstract', False)
+					) and BinderModel in getmro(base)[1:]:
```

Comment:
Is there a reason that getmro is used? Is there a difference between this and `isinstance(base, BinderModel)`? If not, I would suggest to use isinstance for readability

Reply:
Very valid point, I probably was thinking too complicated since I was already using getmro in the line above.